### DISCUSSION TITLE
Modification KDF value

### DISCUSSION THREAD

**Discussion by c8agomez** [dani-garcia/vaultwarden#3889] [General]
After updating vaultwarden the last version,  when a user log in it asks them to modify the kdf value. Can this value be modified globally for all users? I use vaultwarden with docker-compose.

**Comment by BlackDex** [comment]
We can't modify that value on the server side. It needs to be done by the users via the client, since that knows/asks the master password as a plain text value.

**Reply by c8agomez** [reply]
Ok thanks
